frontend/src/metabase/visualizations/echarts/cartesian/option/axis.ts

```diff
@@ -23,6 +23,7 @@ import { CHART_STYLE } from "metabase/visualizations/echarts/cartesian/constants
 
 import type { ChartMeasurements } from "../chart-measurements/types";
 import { isNumericAxis, isTimeSeriesAxis } from "../model/guards";
+import { getAutoAxisEnabledSetting } from "../utils/axis";
```

Comment:
We may want to store the computed rotation in the ChartMeasurements to avoid recomputation

Reply:
Done